documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,50 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application.
+
+In addition to network install you can explicitly boot your device with files downloaded via HTTP with xref:raspberry-pi.adoc#BOOT_ORDER[boot mode] `7`. You can still use this even if xref:raspberry-pi.adoc#NET_INSTALL_ENABLED[network install / keyboard detection on boot is disabled].
+
+You could for example add this to your `BOOT_ORDER` as a fall back boot method or put it behind a GPIO conditional to initiate HTTP download from your own server when a GPIO is pulled low. e.g.
+
+```
+[gpio7=0]
+BOOT_ORDER=0xf7
+HTTP_HOST=downloads.raspberrypi.org
+NET_INSTALL_ENABLED=0
+```
+
+In this case if GPIO 7 were to be pulled low, the files http://downloads.raspberrypi.org:80/net_install/boot.img and http://downloads.raspberrypi.org:80/net_install/boot.boot.sig would be downloaded. If network install / keyboard detection on boot were enabled it would use the same url. If GPIO 7 were not pulled low the behaviour would be unchanged.
+
+HTTP in the `BOOT_ORDER` will be ignored if secure boot is enabled and xref:raspberry-pi.adoc#HTTP_HOST[HTTP_HOST] is not set.
+
+=== Requirements
+
+To use HTTP boot you need to xref:raspberry-pi.adoc#bootloader_update_stable[use the LATEST / STABLE bootloader configuration] and update to a bootloader dated 10th March 2022 or later. HTTP boot only works over ethernet, so you need an Ethernet cable connected to the network, e.g. via a socket on the back of your router.
+
+=== Keys
+
+All HTTP downloads must be signed. The bootloader includes a public key for the files on the default host `fw-download-alias1.raspberrypi.com`. This key will be used to verify the network install image *unless* you set xref:raspberry-pi.adoc#HTTP_HOST[HTTP_HOST] *and* include a public key in the eeprom. This allows you to host the Raspberry Pi network install images on your own server.
+
+WARNING: Using your own network install image will require you to add your public key to the eeprom. At the moment, if you then apply a public eeprom update, your key will be lost and need to be re-added.
+
+USBBOOT has all the tools needed to program public keys, see https://github.com/raspberrypi/usbboot/blob/master/Readme.md
+
+----
+# Add your PUBLIC key to the eeprom. boot.config contains your modifications
+rpi-eeprom-config -c boot.conf -p mypubkey.pem -o pieeprom.upd pieeprom.original.bin
+
+# Generate sig for your eeprom
+rpi-eeprom-digest -i pieeprom.upd -o pieeprom.sig
+
+# Sign with your PRIVATE key the image network install will load
```

Comment:
This sentence feels a bit clunky. Perhaps "Sign the network install image with your PRIVATE key" instead?